cmapi/cmapi_server/handlers/cluster.py

```diff
@@ -160,29 +146,19 @@ def shutdown(
             'Cluster shutdown command called. Shutting down the cluster.'
         )
         operation_start_time = str(datetime.now())
-        if not in_transaction:
-            with TransactionManager():
-                toggle_cluster_state(ClusterAction.STOP, config)
-        else:
-            toggle_cluster_state(ClusterAction.STOP, config)
-
+        toggle_cluster_state(ClusterAction.STOP, config)
```

Comment:
I am missing why don't we want TransactionManager here?

Reply:
Discussed. 
Tramsaction Manager used in a top level calls.